src/styles/components/_button.scss

```diff
@@ -107,8 +107,9 @@
   padding: 0;
 
   table.table & {
-    height: inherit;
-    line-height: inherit;
+    margin: -4px 0 -6px !important;
+    position: relative;
+    top: -2px;
```

Comment:
Kun je deze changes wat meer onderbouwen? In #306 zijn juist de `height` en `line-height` properties toegevoegd. Is de scope van #306 nog steeds gedekt? Heb je naar het Percy screenshot gekeken? https://percy.io/dso-toolkit/dso-toolkit/builds/2030890/view/132010189/1280?browser=chrome&mode=diff

Er zit een klein verschil in, is dit acceptabel/verklaarbaar? @MichielDHVD kijk je mee?

Reply:
`height` en `line-height` bleken dus niet de oplossing. Ik heb het omgebouwd. Check is https://www.dso-toolkit.nl/7.2.0/components/detail/accordion-table.html , met in deze branche de twee verschillende verschijningsvormen van de knop met en zonder border verwerkt.